chainercv/links/connection/convolution_2d_block.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+import chainer
+from chainer.functions import relu
+from chainer.links import BatchNormalization
+from chainer.links import Convolution2D
+
+
+class Convolution2DBlock(chainer.Chain):
+    """Convolution2D --> (Batch Normalization) --> Activation
```

Comment:
> One way to avoid this issue is to separate Conv->ReLU and Conv->BN->ReLU.

It looks good to me. So `BN->ReLU->Conv` will be `BNConvolution2DBlock`?

I think `Convolution2D` is too lengthy and  2D is very common in computer vision. How about `ConvBlock` and `ConvBNBlock`?